telemetry/specs/INT.mdk

```diff
@@ -318,6 +318,10 @@ simply leaves those decisions to device vendors.
   - The build-up of traffic in the queue (in bytes, cells, or packets) that the
 INT packet observes in the device while being forwarded.
 
+* Buffer occupancy
+  - The build-up of traffic in the shared buffer (in bytes, cells, or packets) that the
+INT packet observes in the device while being forwarded.
+
```

Comment:
Perhaps remove the word "shared". Reporting both queue occupancy and buffer occupancy makes sense only if the buffer is shared, but technically the buffer could be a per-queue buffer, just that in this case, both values reported will be the same. INT source does not know whether downstream switches have shared buffers or not. It can set both bits. Switches that have per-queue buffers (unlikely, switches typically use shared buffers) will just report the same value for both instructions.

Reply:
May be word this differently, to say this instruction is to get buffer occupancy. Use-case is when buffer is shared.

Reply:
Removed the reference to shared.

Reply:
Minor editorial rewording:
The use case is when the buffer is shared between multiple queues.

Reply:
Added the minor editorial change suggested.